docs/data-releases.md

```diff
@@ -8,7 +8,7 @@ A DCC data release is a curated data set produced from clinical and experimental
 
 ## Data Release 1.0 
 
-**Release Date: July 30, 2020**
+**Release Date: MM/DD/YYYY**
```

Comment:
should we do YEAR - MONTH - DAY?  That way it can be sorted easier?

Reply:
@rosibaj I"m not sure what you mean by sorted? I was thinking of when the date is known just writing it July 12, 2020...

Reply:
but you changed it to the mm/dd/yyyy format?

Reply:
ya sorry I just changed that so we don't forget to change it from my fake date...any format is fine probably

Reply:
ahh i see ok!

Reply:
I'll remove the dates for now so it's not confusing